.github/workflows/ml-server.yml

```diff
@@ -0,0 +1,25 @@
+name: ml-server
+run-name: ml-server
+on:
+  push:
+    paths:
+      - "ml_server/**"
+      - ".github/workflows/ml-server.yml"
+  pull_request:
+    paths:
+      - "ml_server/**"
+      - ".github/workflows/ml-server.yml"
+jobs:
+  linters:
+    strategy:
+      matrix:
+        python-version: ["3.12.1"]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+          cache: "pip"
+      - run: pip install flake8
```

Comment:
lets install using poetry dev dependencies. feel free to address to other services actions file

Reply:
Every workflow is using this, and I don't want to change the way how it's done for one service.